modules/BuildTools.Tasks/BuildTools.Tasks.csproj

```diff
@@ -9,16 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\**\*.cs" />
```

Comment:
We'd still run into this in the future: https://github.com/aspnet/Coherence-Signed/issues/558